stencil.config.ts

```diff
@@ -29,6 +29,14 @@ export const config: Config = {
     hashFileNames: false,
     namespace: 'lime-elements',
     outputTargets: [targetDist, targetWww],
+    commonjs: {
+        namedExports: {
+            'node_modules/react-dom/index.js': [
+                'render',
+                'unmountComponentAtNode',
+            ],
+        },
+    },
```

Comment:
What does this do?

Reply:
I think it's needed in order to convert the CommonJS modules exported by React to ES6 modules. Rollup was not able to find them otherwise.